core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -0,0 +1,326 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+/**
+ * Configurations properties constants. Please check and update Configuration-Settings.md file when
+ * you change or add Alluxio configuration properties.
+ */
+public final class PropertyKey {
```

Comment:
I feel `Property` also implies both the key and value, whereas here it is only the key part.
Not really feeling strong on `PropertyKey` though,  `Property`  is acceptable too for me